packages/rpc-graphql/src/resolvers/account.ts

```diff
@@ -13,3 +13,19 @@ export const resolveAccount = (fieldName: string) => {
     ) =>
         parent[fieldName] === null ? null : context.loaders.account.load({ ...args, address: parent[fieldName] }, info);
```

Comment:
Maybe this is what you mean by ‘This is remedied in the next commit in the stack,’ but what should happen here is:

1. The account resolver introspects the query itself, and looks into its own subselection for _direct_ child `data` fields.
    a. If there are no data fields, fetch the account with whatever encoding is lightest on the RPC.
    b. If there _are_ data fields, fetch the account with one of those encodings.

Reply:
Yeah, I was going to tackle this in the next commit, but I guess now that I think of it there's no reason for this one then. 😅